test/integration/smoke/test_list_ids_parameter.py

```diff
@@ -194,10 +194,7 @@ def setUpClass(cls):
         cls._cleanup = [
                         cls.disk_offering,
                         cls.account,
-                        cls.service_offering,
-                        cls.snapshot_1,
-                        cls.snapshot_2,
-                        cls.snapshot_3
+                        cls.service_offering
```

Comment:
I agree with your order.  @nvazquez can you update the order to what rohit posted in his commit.  @rhtyd is there a reason you added this to your PR?  Will you just remove it from that PR once this is updated and merged?